src/lib/stores/orderStore.ts

```diff
@@ -0,0 +1,102 @@
+import { PUBLIC_PB_HOST } from "$env/static/public";
+import { writable } from "svelte/store";
+import pocketbase from "pocketbase";
+import { Order } from "$lib/types";
+import type { State } from "$lib/types";
+
+// ref: https://github.com/pocketbase/js-sdk?tab=readme-ov-file#nodejs-via-npm
+import eventsource from "eventsource";
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+(global as any).EventSource = eventsource;
+
+const pb = new pocketbase(PUBLIC_PB_HOST);
```

Comment:
Hvorfor initialisere en ny pocketBase instans her i stedet for å bruke den som finnes fra før i `authStore`? Hvis jeg forstår riktig skal denne store-en brukes client-side, så det bør være bedre å bruke den.

Reply:
Siden `authStore` er en `store` og eksporterer `pb` valgte jeg å være konsekvent, siden `orderStore` også er en `store`.

En bedre praksis er å definere `pb` som singleton et annet sted enn en `svelte` store. Jeg valgte å ikke gjøre det i denne PR.

Reply:
jeg kommer til å flytte denne ut til en singleton for å se om det har noe effekt på pocketbase problemer